flink-kubernetes-shaded/pom.xml

```diff
@@ -88,6 +103,7 @@ under the License.
                                         <exclude>META-INF/DEPENDENCIES</exclude>
                                         <exclude>META-INF/LICENSE</exclude>
                                         <exclude>META-INF/MANIFEST.MF</exclude>
+                                        <exclude>org/apache/flink/kubernetes/shaded/org/yaml/snakeyaml/**</exclude>
```

Comment:
Wouldn't this line be the only change we need for `flink-kubernetes-shaded`?

Reply:
I am not sure whether it is possible to update the bundled `NOTICE` file to also remove the `org.yaml:snakeyaml:1.27`.

Reply:
@wangyang0918 
The NOTICE has this and only this relevant entry:
```
  - SnakeYAML (https://bitbucket.org/snakeyaml/snakeyaml) org.yaml:snakeyaml:bundle:1.32
    License: Apache License, Version 2.0  (http://www.apache.org/licenses/LICENSE-2.0.txt)
```

@gyfora Because of the relocation, I added comments to make it more clear.

Reply:
What I mean is we still have `org.yaml:snakeyaml:1.27` in the NOTICE file of `flink-kubernetes-shaded-1.2-SNAPSHOT.jar` even after this PR. Right?

Reply:
Yes that is correct, @mbalassi didnt find a straightforward way to do this but this should have no release implications as we correctly indicate the new version in the other notice file.